languages/go/exercises/concept/booleans/booleans.go

```diff
@@ -0,0 +1,22 @@
+package booleans
+
+//CanFastAttack can be executed only when the knight is sleeping
```

Comment:
I'd prefer a space after the comment slashes. Not sure which editor you are using but it might be a setting you can change for the editor to automatically add spaces there. In Goland there definitely is a setting.

Reply:
I'm using VS Code and it looks like something went wrong with my formatting tool. I've corrected this now.